src/index.tsx

```diff
@@ -1,18 +1,15 @@
 import { lazy, Suspense } from "react";
-import { createRoot } from "react-dom/client";
+import type { ReactElement } from "react";
 import { ErrorBoundary } from "react-error-boundary";
 
 const App = lazy(async () => import("./App"));
-const containerId = "root";
-const container = document.getElementById(containerId);
-if (!container) {
-  throw Error(`couldn't find element with id ${containerId}!`);
+
+export default function Index(): ReactElement {
+  return (
+    <ErrorBoundary fallback={<div>shit hit the fan bruv!</div>}>
```

Comment:
what the hell? That doesn't belong there. Use a simple component that shows the error message. That should take more than 15 minutes.

Reply:
fixed